contracts/incentives/ERC20TokenIncentivesController.sol

```diff
@@ -0,0 +1,202 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.5;
+pragma experimental ABIEncoderV2;
+
+import {SafeERC20} from '@aave/aave-stake/contracts/lib/SafeERC20.sol';
+import {SafeMath} from '../lib/SafeMath.sol';
+import {DistributionTypes} from '../lib/DistributionTypes.sol';
+import {VersionedInitializable} from '@aave/aave-stake/contracts/utils/VersionedInitializable.sol';
+import {DistributionManager} from './DistributionManager.sol';
+import {IERC20} from '@aave/aave-stake/contracts/interfaces/IERC20.sol';
+import {IScaledBalanceToken} from '../interfaces/IScaledBalanceToken.sol';
+import {IAaveIncentivesController} from '../interfaces/IAaveIncentivesController.sol';
+
+/**
+ * @title ERC20TokenIncentivesController
+ * @notice Distributor contract for rewards to the Aave protocol, using a regular ERC20 token as rewards asset.
+ * @author Aave, Joey Santoro
+ **/
+contract ERC20TokenIncentivesController is
+  IAaveIncentivesController,
+  VersionedInitializable,
+  DistributionManager
+{
+  using SafeMath for uint256;
+  using SafeERC20 for IERC20;
+
+  uint256 public constant REVISION = 1;
+
+  /// @inheritdoc IAaveIncentivesController
+  address public immutable override REWARD_TOKEN;
```

Comment:
Possible change: Keep it as the following, maybe with another name to do not cause confussion
`IERC20 public immutable TOKEN;`

And recover the function:
```
  /// @inheritdoc IAaveIncentivesController
  function REWARD_TOKEN() external view override returns (address) {
    return address(TOKEN);
  }
```